Services/CO.CDP.Organisation.WebApi/UseCase/AssignIdentifierUseCase.cs

```diff
@@ -14,7 +14,22 @@ public class IdentifierSchemes
         public const string Vat = "VAT";
         public const string CompaniesHouse = "GB-COH";
     }
-
+    public static IDictionary<string, string> IdentifierSchemesUK = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase)
```

Comment:
can you add the spaces to make the codes readable 

"GB-COH" => "Companies house number",
            "GB-CHC" => "Charity commission for England & Wales number",
            "GB-SC" => "Scottish charity regulator",
            "GB-NIC" => "Charity commission for northern Ireland number",
            "GB-MPR" => "Mutuals public register number",
            "GG-RCE" => "Guernsey registry number",
            "JE-FSC" => "Jersey financial services commission registry number",
            "IM-CR" => "Isle of Man companies registry number",
            "GB-NHS" => "National health service organisations registry number",
            "GB-UKPRN" => "UK register of learning provider number",
            "VAT" => "VAT number",
            "GB-PPON" => "Ppon",
            "OTHER" => "Other / None",